CMakeLists.txt

```diff
@@ -157,6 +157,7 @@ if(DEFINED ENV{LUA_COMPAT53_DIR})
 endif()
 include_directories(${LUA_COMPAT53_DIR}/c-api)
 
+if (BUILD_MODULE)
```

Comment:
Under this condition, exactly the same procedure goes for `libluv` target. Perhaps it's possible to use a `for tmp_target in ["libluv" , "luv"]` instead, but I'm not familiar enough with Cmake yet to write something like that.